contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -178,6 +178,7 @@
           content = formulaHtmlToMd(content);
           content = imagesHtmlToMd(content);
           content = content.replaceAll('&nbsp;', ' ');
+          content = content.replaceAll('<br>', '\n');
```

Comment:
@MisRob It occurred to me but I haven't taken the time to try it yet since technically it's still the weekend for me:  I wonder if we need to do something like `content = Converter.toMarkdown(content)` here instead?